app/controllers/admin/api/metrics_base_controller.rb

```diff
@@ -1,24 +1,13 @@
 class Admin::Api::MetricsBaseController < Admin::Api::ServiceBaseController
+  include MetricParams
 
   protected
 
-    def metric
-      raise NotImplementedError.new "#{self.class.name} should implemented metric method"
-    end
+  def metric
+    raise NotImplementedError.new "#{self.class.name} should implemented metric method"
```

Comment:
NotImplementedError is not the correct class. It has nothing to do with system calls.

Reply:
This file was touched in a different PR whose commits are included here (https://github.com/3scale/porta/pull/1778). Even so, this specific line only appears in the diff because of indentation fix.